crates/subspace-service/src/lib.rs

```diff
@@ -491,14 +491,9 @@ where
 
             let (node, mut node_runner) = create_dsn_instance::<Block, _>(
                 config.clone(),
-                piece_cache.clone(),
+                piece_cache,
                 root_block_cache.clone(),
-            )
-            .instrument(tracing::info_span!(
-                sc_tracing::logging::PREFIX_LOG_SPAN,
-                name = "DSN"
-            ))
-            .await?;
```

Comment:
You can still instrument that though:
```rust
tracing::info_span!(
    sc_tracing::logging::PREFIX_LOG_SPAN,
    name = "DSN"
)
.in_scope(move || /* do stuff */)?;
```

Reply:
Thanks for reminding. I don't need to instrument that one anymore, but I do need to instrument the inner function, will push extra commit now/

Reply:
Actually, no, it is not necessary. Why would I instrument a synchronous function? It was only necessary before because it was async.